changelog/README.md

```diff
@@ -8,6 +8,10 @@ For more details, see [Environments](../guidelines/environments.md).
 
 # Changelog
 
+## 8th September 2022
+
+* Extended [Customers Get All](../operations/customers.md#get-all-customers) response with `CarRegistrationNumber` field.
```

Comment:
It's referenced like this in the documentation
```suggestion
* Extended [Get all customers](../operations/customers.md#get-all-customers) response with `CarRegistrationNumber` field.
```
![image](https://user-images.githubusercontent.com/1150547/189151913-e0c049f7-1081-41d6-8755-fda38f81f33d.png)

Reply:
Thank you, fixed